mod1/mod11/src/file-3.xoo

```diff
@@ -0,0 +1,11 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// dummy line 7
+// dummy line 8
+// dummy line 9
+// dummy line 10
+// violate HasTag rule: xoo on line 11
```

Comment:
###AWBaPEX9RYfhTR3nVKgk###
sample:mod1/mod11/src/file-3.xoo:BRANCH:feature1@11
Has Tag

Reply:
===RESOLVED===
This issue has been resolved by a user in SonarQube